app/javascript/components/greeting.js

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { fetchGreeting } from "../redux/greetingSlice";
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const greeting = useSelector((state) => state.greeting);
+
+  useEffect(() => {
+    dispatch(fetchGreeting());
+  }, [dispatch]);
+
+  return (
+    <div>
+      <h1>If you want to see diferent greeting, than refresh the page 😊.</h1>
+      <ul>
+        {greeting.map((greeting) => (
+          <li key={greeting.id}>{greeting.greeting}</li>
+        ))}
```

Comment:
* As pointed out by the previous reviewer your API works fine only it is not connected to your react properly therefore is doesn't show any random greetings Kindly fix this 🙏

Reply:
@Davidosky007 
I tried everything I know, but nothing worked. Can you please help with that?